From: Daniel Boles <dboles.src@gmail.com>
Date: Sun, 11 Jun 2023 11:22:45 +0100
Subject: ScrolledWindow: Frame on viewport is not optional,

and seems always to default to true, so don't mention it being optional.

Origin: 3.24.39, commit:c9f0b0d29b882326b4858b4d9fa62291dccfcf43
---
 gtk/gtkscrolledwindow.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gtk/gtkscrolledwindow.c b/gtk/gtkscrolledwindow.c
index a931d2b..f4c09d0 100644
--- a/gtk/gtkscrolledwindow.c
+++ b/gtk/gtkscrolledwindow.c
@@ -56,9 +56,9 @@
  * @Title: GtkScrolledWindow
  * @See_also: #GtkScrollable, #GtkViewport, #GtkAdjustment
  *
- * GtkScrolledWindow is a container that accepts a single child widget, makes
+ * GtkScrolledWindow is a container that accepts a single child widget and makes
  * that child scrollable using either internally added scrollbars or externally
- * associated adjustments, and optionally draws a frame around the child.
+ * associated adjustments.
  *
  * Widgets with native scrolling support, i.e. those whose classes implement the
  * #GtkScrollable interface, are added directly. For other types of widget, the
